quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/rest/DashboardResourceAssembler.java

```diff
@@ -0,0 +1,30 @@
+package com.github.reflectoring.infiniboard.quartermaster.dashboard.rest;
+
+import org.springframework.hateoas.mvc.ResourceAssemblerSupport;
+import org.springframework.stereotype.Component;
+
+import com.github.reflectoring.infiniboard.quartermaster.dashboard.domain.Dashboard;
+import com.github.reflectoring.infiniboard.quartermaster.widget.rest.WidgetController;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+@Component
+public class DashboardResourceAssembler extends ResourceAssemblerSupport<Dashboard, DashboardResource> {
+
+    public DashboardResourceAssembler() {
+        super(DashboardController.class, DashboardResource.class);
+    }
+
+    @Override
+    public DashboardResource toResource(Dashboard entity) {
+        DashboardResource resource = new DashboardResource();
+        resource.setName(entity.getName());
+        resource.setDescription(entity.getDescription());
+        resource.add(
+                linkTo(methodOn(DashboardController.class).getDashboard(entity.getId())).withRel("self"));
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)